test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/BarnimReleaseIT.java

```diff
@@ -67,10 +67,7 @@ public void correctUnitRegistrations() throws Exception {
         assertEquals(1, LogAssert.count(simulationRule, "output.csv",
                 ".*RSU_REGISTRATION;.*"
         ));
-        assertEquals(53, LogAssert.count(simulationRule, "output.csv",
-                ".*TRAFFICLIGHT_REGISTRATION;.*"
-        ));
-        assertEquals(53, LogAssert.count(simulationRule, "output.csv",
+        assertEquals(42, LogAssert.count(simulationRule, "output.csv",
```

Comment:
Is this due to the new network representation?

Reply:
Jep, some traffic lights are gone :<